config.py

```diff
@@ -0,0 +1,43 @@
+import os
+
+
+class Config(object):
+    """
+    Common configurations
+    """
+
+    DEBUG = False
+    SQLALCHEMY_TRACK_MODIFICATIONS = False
+    SECRET_KEY = os.environ.get('SECRET_KEY')
+    FLASKY_POSTS_PER_PAGE = 4
+
+
+class DevelopmentConfig(Config):
+    """
+    Development configurations
+    """
+
+    SQLALCHEMY_DATABASE_URI = os.environ.get('DB_URL')
+    DEBUG = True
+    SQLALCHEMY_ECHO = True
+
+
+class TestingConfig(Config):
+    """Configurations for Testing, with a separate test database."""
+    TESTING = True
+    SQLALCHEMY_DATABASE_URI = os.environ.get('TESTDB')
+
+
+class ProductionConfig(Config):
+    """
+    Production configurations
+    """
+    DEBUG = False
```

Comment:
I think here you are repeating your self
`DEBUG = False`
is already defined in Config class.
There is no need of re-defining it here since the this class inherits from the Config class and DEBUG has the same value in both classes

Reply:
Noted, I hadn't noticed that, I will update it, Thanks